assets/js/modules/analytics/components/common/PropertySelect.js

```diff
@@ -68,7 +68,7 @@ export default function PropertySelect() {
 		}
 	}, [ propertyID, selectProperty ] );
 
-	if ( ! isValidAccountID( accountID ) ) {
+	if ( ! isValidAccountSelection( accountID ) ) {
```

Comment:
This change isn't really necessary since we don't render any of these selects when account is set to CREATE but, it's consistent with the others so I suppose it's fine 👍